content/concepts/kata/translations.md

```diff
@@ -49,9 +49,10 @@ _*WARNING:* Currently, "Coauthor" flag seems to be affected by a bug which preve
 ## Pending translations
 
 When translation is published, it's put into a pending state and awaits for review and approval. Sufficiently privileged users can search for them, review them, and request some fixes, approve, or reject them. Currently, pending translations can be found in one of following ways:
```

Comment:
"When translation is published" --> "When a translation is published"

Reply:
```suggestion
When a translation is published, it's put into a pending state and awaits for review and approval. Sufficiently privileged users can search for them, review them, and request some fixes, approve, or reject them. Currently, pending translations can be found in one of following ways:
```